### DISCUSSION TITLE
Deploy music-bot example to Heroku

### DISCUSSION THREAD

**Discussion by LorenzoGala** [discordjs/voice#216] [Q&A]
Hello everyone and thank you for the amazing work on Discord Voice. I'm looking for help because I don't known what to code inside the Procfile for Heroku considering that I have to npm install dependencies from two different folders (main one and music-bot example one)

**Comment by ncjsvr** [comment]
If everything is set up correctly in your package.json file, your Procfile just needs to include the following:

`worker: npm start`

Then you disable the web dyno and enable the worker dyno.

EDIT: This should help you, found on Google: https://elements.heroku.com/buildpacks/synicalsyntax/discord.js-heroku
